test/controllers/api/v2/errata_controller_test.rb

```diff
@@ -127,6 +127,20 @@ def test_index_with_available_for_content_view_version_protected
       end
     end
 
+    def test_index_with_org_search
+      test_repo2 = Repository.find(katello_repositories(:fedora_17_x86_64).id)
+      test_repo2.write_attribute(:environment_id, katello_environments(:organization1_library).id)
```

Comment:
Thanks for the test. I see it fails without your change which is perfect.

I'm curious about `write_attribute` - we don't use that anywhere else in the code so I'm wondering if you're making a case for using it vs `test_repo2.environment = katello_environments(:organization1_library)`

I prefer the latter because it keeps us more in our world rather than rails world by using an underlying API